File: documentation-examples-kotlin/src/kotlin/dev/typr/foundationskt/docs/sqlserver/IntegerTypes.kt
package dev.typr.foundationskt.docs.sqlserver

import dev.typr.foundationskt.*
import dev.typr.foundationskt.data.*

@Suppress("unused")
class IntegerTypes {
    //start
    val tinyType: SqlServerType<Uint1> = SqlServerTypes.tinyint   // Note: unsigned!
    val intType: SqlServerType<Int> = SqlServerTypes.int_
    val bigType: SqlServerType<Long> = SqlServerTypes.bigint
    //stop
}

File: documentation-examples-scala/src/scala/dev/typr/foundationssc/docs/sqlserver/IntegerTypes.scala
package dev.typr.foundationssc.docs.sqlserver
import dev.typr.foundationssc.*
import dev.typr.foundationssc.data.*



@SuppressWarnings(Array("unused"))
object IntegerTypes:
  //start
  val tinyType: SqlServerType[Uint1] = SqlServerTypes.tinyint   // Note: unsigned!
  val intType: SqlServerType[Int] = SqlServerTypes.int_
  val bigType: SqlServerType[Long] = SqlServerTypes.bigint
  //stop
